.github/workflows/cd-beta.yaml

```diff
@@ -77,12 +81,25 @@ jobs:
   #           SLACK_USERNAME: Deploy Bot
   #           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
 
+  # Delete the stage branch after it is merged into beta
+  delete_stage_branch:
+    if: github.head_ref == 'stage'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Delete stage branch
+        run: git push origin :stage
+
   create_stable_pr:
     runs-on: ubuntu-latest
-    # needs: [deploy_beta]
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
+
+      - name: Ensure origin/stable branch exists
+        if: git branch -r --list origin/stable != 'origin/stable'
+        run: it checkout beta && git switch --create stable && git push -u origin stable
```

Comment:
Not sure if this is how it's supposed to be but it seems like it's missing a `g` in the git checkout

Reply:
Good catch! Thanks @finnar-bin